### ISSUE TITLE
Check for missing translations due to JSDoc dependency issue, fix any that exist

### ISSUE THREAD

**Issue by jbphet** [phetsims/rosetta#279]
In https://github.com/phetsims/chipper/issues/1189, a problem was reported and then fixed in a maintenance release where builds of some sims were failing for a period of time.  If translations were submitted during the time period during which these sims were broken, the sims would not have been built and deployed.  The build logs should be checked for failures and, if there are some and if the translation is indeed missing from the website, a rebuild and deployment should be initiated.

The list of affected translations can be found [here](https://github.com/phetsims/chipper/issues/1189#issuecomment-1026321711).

The window of time where this problem existed was from Jan 21 2022 through Jan 31 2022.

**Comment by jbphet** [comment]
I did an initial scan through the logs for the time period in question, and I'm seeing what looks like 211 build requests and 13 failures.  Ideally, we would check each of the failure cases to see if there truly is no translation for that sim from later than that date, since a translator could have re-submitted it.  I'm working with the QA team to see if they can help out with this process.

**Comment by jbphet** [comment]
I talked with @kathy-phet and @KatieWoe, and we decided that I should upload a filtered version of the build server log from the time period in question, meet with @KatieWoe to discuss, and she will take it from there.  She will then let me know which sims need to be rebuilt.

I trimmed a copy of the build log to only have the days during the period, then used this command to filter it:
```
grep "deploy request received\|simName:\|locales:\|BUILD ABORTED\|finished successfully" build-server-log-for-broken-window.txt > temp.txt
```

I'll attach the results.
[temp.txt](https://github.com/phetsims/rosetta/files/8073916/temp.txt)

**Comment by KatieWoe** [comment]
- [ ] Sim = build-a-molecule, Locale = ko, Day and time = Jan 23 01:25:01
- [ ] Sim = build-a-molecule, Locale = ko, Day and time = Jan 23 05:11:27
- [ ] Sim = density, Locale = de, Day and time = Jan 23 08:20:58
- [ ] Sim = circuit-construction-kit-ac, Locale = ar, Day and time = Jan 23 09:27:53
- [ ] Sim = circuit-construction-kit-ac, Locale = sk, Day and time = Jan 23 14:18:49
- [ ] Sim = circuit-construction-kit-ac-virtual-lab, Locale = sk, Day and time = Jan 23 14:23:23
- [ ] Sim = density, Locale = sk, Day and time = Jan 23 15:03:05
- [ ] Sim = density, Locale = ms, Day and time = Jan 23 23:27:37
- [ ] Sim = circuit-construction-kit-ac, Locale = ar, Day and time = Jan 24 00:49:54
- [ ] Sim = density, Locale = is, Day and time = Jan 24 06:43:20
- [ ] Sim = fourier-making-waves, Locale = is, Day and time = Jan 24 08:08:11
- [ ] Sim = energy-skate-park, Locale = hy, Day and time = Jan 28 07:58:13

**Comment by kathy-phet** [comment]
@jbphet @KatieWoe - I noticed that 2 sim translations are on are the rebuild list. However Fourier Making Waves isn't on the list of affected sims that JO has here: 
https://github.com/phetsims/chipper/issues/1189#issuecomment-1026321711

@jbphet - Do you understand that inconsistency?

**Comment by jbphet** [comment]
> @jbphet - Do you understand that inconsistency?

Partially, but I think we'd need to bring in @jonathanolson to fully understand it.  The build log _does_ show a failed build of an Icelandic translation of fourier-making-waves during that time period, and the failure message looks like the same one that was occurring for the sims that _are_ on the list.  That message is:

```
error: BUILD ABORTED! Error: npm update in ../chipper failed with exit code 1
```

I'll assign @jonathanolson to see if he can tell us why the sim would be having failed builds during this period and yet not be on his list in https://github.com/phetsims/chipper/issues/1189#issuecomment-1026321711.

Ultimately, my action for this will be the same, which is to try to get this sim and locale published.

**Comment by jbphet** [comment]
I built the Icelandic version of Fourier a while back, and it appears to have worked, since that one is now live on the site at https://phet.colorado.edu/sims/html/fourier-making-waves/latest/fourier-making-waves_is.html.

I tried building the Korean version of Build a Molecule, but it failed because there is no translation file for this.  In other words, the file `build-a-molecule_ko.json` does not exist in the babel repo.  In all likelihood this is due to the problems reported in https://github.com/phetsims/rosetta/issues/262.

I think we've done all that we can here, and in my opinion it's not worth @jonathanolson's time to answer the (question raised above)[[xxxxx](https://github.com/phetsims/rosetta/issues/279#issuecomment-1058499458)] since I've rebuilt the sim, so I'm going to close this.
